slick.grid.js

```diff
@@ -6121,6 +6133,9 @@ if (typeof Slick === "undefined") {
       "getFrozenRowOffset": getFrozenRowOffset,
       "setColumnHeaderVisibility": setColumnHeaderVisibility,
       "sanitizeHtmlString": sanitizeHtmlString,
+      "isViewportHasVScroll": isViewportHasVScroll,
```

Comment:
I usually use `is...` for a flag rarely for a method name, what do you think about `hasViewportVScroll` and `hasViewportHScroll`?

Reply:
yep I think `has...` is more suitable in such case. Let me rename it.

Reply:
What do you think about:
```diff
 // in slick.grid.js
 // ...
 // exist function to get scrollbar dimensions
 function getScrollbarDimensions() {
   return scrollbarDimensions;
 }
-function isViewportHasHScroll() {
-  return viewportHasHScroll
-}
-
-function isViewportHasVScroll() {
-  return viewportHasVScroll;
-}
+function getDisplayedScrollbarDimensions() {
+  return {
+    width: viewportHasVScroll ? scrollbarDimensions.width : 0,
+    height: viewportHasHScroll ? scrollbarDimensions.height : 0
+  };
+}
```